specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2025-04-01/containerregistry.json

```diff
@@ -344,21 +341,15 @@
         ],
         "description": "Creates a connected registry for a container registry with the specified parameters.",
         "operationId": "ConnectedRegistries_Create",
-        "consumes": [
```

Comment:
why do we need to remove `consumes` and `produces` in connected registry related api? I don't set the change in other resource apis.